_posts/2019-07-03-reinforce-2019-recap.md

```diff
@@ -0,0 +1,104 @@
+
+## SDD324 Setting up a DevSecOps pipeline to automate vulnerability scanning of Docker images
+A builders session where most of the one hour's time was used waiting for CloudFormation to create stacks. CoreOS Clair and Klar were used for scanning Docker images for known vulnerabilities and added as a step to a CodePipeline setup. [Link to materials](https://github.com/aws-samples/aws-codepipeline-docker-vulnerability-scan)
+
+![SDD324](/img/reinforce-2019-recap/day2-sdd324.jpg)
+
+## SEJ-001 - Security Jam
+This "hackathon" was a bit different compared to re:Invent's Game Day. This was basically similar to the security jam lounge, you had 9 different challenges that you tried to solve as a team in 4 hours. The topics in the challenges were quite varying, everything from IAM to IoT to API Gateway was used. Quite an intense session, luckily I had some talented people in my team so we managed to get 8 of the 9 challenges done and got the first place! We were also the first team to solve a sponsored challenge so we also won some Trend Micro swag and Amazon.com giftcards.
+
+![SEJ-001](/img/reinforce-2019-recap/day2-securityjam.jpg)
+
+## SDD-326 Security best practices for Amazon EKS
+Another builders session where one hour was clearly a too short time. Setting up an EKS cluster and tooling from scratch via Cloudformation took almost the whole time, so we didn't get to the actual point of the session (using IAM credentials for controlling access to pods). [Link to materials](https://github.com/meyjames/EKSKube2IAM)
+
+![SDD-326](/img/reinforce-2019-recap/day2-sdd326.jpg)
+
+## Closing reception
+
+After the second day, there was an official closing reception just outside the conference center with plenty of outdoor games, music, food, drinks and a chill atmosphere.
+
+![Closing party](/img/reinforce-2019-recap/closingreception.jpg)
+
+# Conclusion
+
+Overall, everything was well organized and a lot of interesting stuff was happening. Two days almost felt too short for an event like this, as they could've easily had more repeats of the sessions and run the event for a third day. A lot of hands on learning this time around, as I tried to choose many workshops and builders session. I learned a lot about some AWS services that I haven't used that much in my projects during these two days. Hackathons and jams were again perhaps the highlight.
+
+![A Cloud Guru](/img/reinforce-2019-recap/cloudguru.jpg)
+
+Was the event worth attending? Well, obviously you aren’t going to get quite as much and as varied topics in a two day conference as you would in a week long re:Invent. But still, security is such an integral part of pretty much everything that AWS does, so it was actually quite interesting to look at all the services from a different viewpoint than you would perhaps normally look at them. From new announcements point of view, the event was quite lackluster, but perhaps they are saving some feature or product announcements for re:Invent.
```

Comment:
`it was actually quite interesting to look at all the services from a different viewpoint than you would perhaps normally look at them.` some repetition in the end.